packages/fetchai/connections/p2p_libp2p/check_dependencies.py

```diff
@@ -19,12 +19,35 @@
 # ------------------------------------------------------------------------------
 
 """Check that the dependencies 'gcc' and 'go' are installed in the system."""
+import asyncio
 import re
 import shutil
 import subprocess  # nosec
+import sys
+from distutils.dir_util import copy_tree
 from itertools import islice
 from typing import Any, Iterable, List, Pattern, Tuple
 
+from aea.helpers.base import ensure_dir
+
+
+try:
+    # flake8: noqa
+    # pylint: disable=unused-import,ungrouped-imports
+    from .connection import (  # type: ignore
```

Comment:
Use full path: `from packages.fetchai.connections....`

Reply:
will stay it as is